packages/marketplace-ui/src/components/admin/TokenLaunches.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  Text,
+  Flex,
+} from "@chakra-ui/react";
+import { PublicKey } from "@solana/web3.js";
+import { truthy, useFungibleChildEntangler, useStrataSdks, useTokenBondingFromMint } from "@strata-foundation/react";
+import { LaunchPreview } from "./LaunchPreview";
+import React, { useState } from 'react';
+import { useAsync } from "react-async-hook";
+import { FungibleEntangler, IFungibleChildEntangler } from "@strata-foundation/fungible-entangler";
+import { useWallet } from "@solana/wallet-adapter-react";
+
+interface TokenPreviewProps {
+  mintKey: PublicKey | undefined;
+  name: string | undefined;
+  image: string | undefined;
+}
+
+export const TokenLaunches = ({ mintKey, name, image }: TokenPreviewProps) => {
+  const { info: tokenBonding } = useTokenBondingFromMint(mintKey);
+  const { fungibleEntanglerSdk } = useStrataSdks();
+  const { publicKey } = useWallet();
+  const [parsed, setParsed] = useState<PublicKey[]>([]);
+  const { info: entangler } = useFungibleChildEntangler(parsed[0]);
+  useAsync(async () => {
+    if (!mintKey || !fungibleEntanglerSdk) return
+    const parentAccounts = await fungibleEntanglerSdk?.provider.connection.getProgramAccounts(FungibleEntangler.ID,
```

Comment:
I'll make sure you can use vybe for this by EOD. Also, can you make sure this supports if the token is fully managed?

Reply:
yeah FM tokens work fine. will resolve this once vybe is set up